### PR TITLE
Fix unresponsive button.

### PR DESCRIPTION
**Pull request by cheikhshift**
Once in aggregate view you cannot change the data viewing mode.

### CODE REVIEW

File: traceapp/tmpl/data/aggregate.html [hunk 1]
```diff
@@ -24,10 +24,14 @@
...
   }
+	.clickable-z-index {
+		z-index: 1;
+    		position: relative;	
+	}
 </style>
 
 <!-- View Mode menu -->
-<div class="btn-group pull-right" role="group" aria-label="..." id="top-right-btns">
+<div class="btn-group pull-right  clickable-z-index" role="group" aria-label="..." id="top-right-btns">
```
Comment (slimsag) [inline traceapp/tmpl/data/aggregate.html]: Looks like you have an extra space here

File: traceapp/tmpl/data/aggregate.html [hunk 2]
```diff
@@ -24,10 +24,14 @@
     position: relative;
     top: -100px;
   }
+	.clickable-z-index {
+		z-index: 1;
+    		position: relative;	
+	}
```
Comment (slimsag) [inline traceapp/tmpl/data/aggregate.html]: the indentation is off here

**Comment by slimsag** [comment]
You'll need to run `go generate sourcegraph.com/sourcegraph/appdash/traceapp/tmpl` and commit the result as well, since the HTML is embedded.

**Comment by cheikhshift** [comment]
Will do!

**Comment by slimsag** [comment]
Closing in favor of https://github.com/sourcegraph/appdash/pull/195

**Pull request closed without merging**